src/components/TableData/index.js

```diff
@@ -0,0 +1,131 @@
+import * as React from "react";
```

Comment:
This Table component should be a common component, We should make use of react-table library, parent component will pass the columns and data, this table will have the responsibility of rendering the view only.